### DISCUSSION TITLE
Domain Specific setting for white background setting

### DISCUSSION THREAD

**Discussion by tcchuin** [plateaukao/einkbro#204] [Ideas]
Hi team,

It would be good to have white background memory/setting specific to a domain.
For example some website works well with the setting, but for some website (e.g. twitter mobile) it's not usable at all

Will be good if it's possible for the browser to remember which to enable/disable it on

Thanks!

**Comment by Cwpute** [comment]
Also have domain specifics settings for Reader mode as well would be wonderful.
